elliptic-curve/src/hash2curve/group_digest.rs

```diff
@@ -1,17 +1,16 @@
 use super::MapToCurve;
 use crate::{
     hash2field::{hash_to_field, ExpandMsg, FromOkm},
-    Result,
+    ProjectiveArithmetic, Result,
 };
 use group::cofactor::CofactorGroup;
 
 /// Adds hashing arbitrary byte sequences to a valid group element
-pub trait GroupDigest {
+pub trait GroupDigest: ProjectiveArithmetic<ProjectivePoint = Self::Output> {
```

Comment:
This looks like `hash2curve` feature is now dependent on the `arithmetic` feature in Cargo.toml.

CI isn't catching it because we don't currently test the crate builds with `hash2curve` alone enabled. We need to add it to the list here:

https://github.com/RustCrypto/traits/blob/master/.github/workflows/elliptic-curve.yml#L45

And it would be good to add it here too:

https://github.com/RustCrypto/traits/blob/master/.github/workflows/elliptic-curve.yml#L53

Reply:
I'd suggest updating the CI config to confirm this, and if it breaks the build, adjusting this line in Cargo.toml:

https://github.com/RustCrypto/traits/blob/master/elliptic-curve/Cargo.toml#L50

....and changing it to:

```toml
hash2curve = ["arithmetic", "digest"]
```

Reply:
This constraint was only necessary to implement `hash_to_scalar`, I will re-add it later when I've figured it out.